File: src/Entites/Event.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entites;

import static com.sun.org.apache.xalan.internal.lib.ExsltDatetime.date;
import static com.sun.org.apache.xalan.internal.lib.ExsltDatetime.date;
import static com.sun.org.apache.xalan.internal.lib.ExsltDatetime.date;
import static com.sun.org.apache.xalan.internal.lib.ExsltDatetime.date;
import java.sql.Date;

/**
 *
 * @author ayoub
 */
public class Event {
      private int id;
    private String nom_event;
    private String type_sport;
    private int phone_number;
    private Date datee;
    private String location;
    
     public Event(){
        
    }
    public Event(String nom_event ,String type_sport, int phone_number, Date datee, String location){
        this.nom_event=nom_event;
        this.type_sport=type_sport;
        this.phone_number=phone_number;
        this.datee=datee;
        
        this.location=location;
       
    }
      public Event(int id, String nom_event, String type_sport, int phone_number, Date datee, String location) {
        this.id=id;
        this.nom_event=nom_event;
        this.type_sport=type_sport;
        this.phone_number=phone_number;
        this.datee=datee;
        
        this.location=location;
          

  
  
     }
      
    public String getnom_event (){return nom_event;}
    public void setNom_event (String nom_event){ this.nom_event = nom_event;}
    
    public String gettype_sport (){return type_sport;}
    public void settype_sport (String type_sport){ this.type_sport = type_sport;}
    
    public int getphone_number (){return phone_number;}
    public void setPhone_number (int phone_number){ this.phone_number = phone_number;}
    
   public Date getdatee (){return datee;}
    public void setdatee (Date datee){ this.datee = datee;}
    
    
    
    
    public String getLocation (){return location;}
    public void setLocation (String location){ this.location = location;}
    
    
  
     
}
    
    

File: src/Entity/Event.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Event
 *
 * @ORM\Table(name="event")
 * @ORM\Entity
 */
class Event
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="nom_event", type="string", length=255, nullable=false)
     */
    private $nomEvent;

    /**
     * @var string
     *
     * @ORM\Column(name="type_sport", type="string", length=255, nullable=false)
     */
    private $typeSport;

    /**
     * @var string
     *
     * @ORM\Column(name="phone_number", type="string", length=255, nullable=false)
     */
    private $phoneNumber;

    /**
     * @var string
     *
     * @ORM\Column(name="datee", type="string", length=255, nullable=false)
     */
    private $datee;

    /**
     * @var string
     *
     * @ORM\Column(name="location", type="string", length=255, nullable=false)
     */
    private $location;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNomEvent(): ?string
    {
        return $this->nomEvent;
    }

    public function setNomEvent(string $nomEvent): self
    {
        $this->nomEvent = $nomEvent;

        return $this;
    }

    public function getTypeSport(): ?string
    {
        return $this->typeSport;
    }

    public function setTypeSport(string $typeSport): self
    {
        $this->typeSport = $typeSport;

        return $this;
    }

    public function getPhoneNumber(): ?string
    {
        return $this->phoneNumber;
    }

    public function setPhoneNumber(string $phoneNumber): self
    {
        $this->phoneNumber = $phoneNumber;

        return $this;
    }

    public function getDatee(): ?string
    {
        return $this->datee;
    }

    public function setDatee(string $datee): self
    {
        $this->datee = $datee;

        return $this;
    }

    public function getLocation(): ?string
    {
        return $this->location;
    }

    public function setLocation(string $location): self
    {
        $this->location = $location;

        return $this;
    }


}
